src/server/server.ts

```diff
@@ -980,7 +980,7 @@ namespace ts.server {
     };
 
     const ioSession = new IOSession(options);
-    process.on("uncaughtException", err => {
+    process.on("uncaughtException", (err: any) => {

```

Comment:
why?

Reply:
Because it was failing compilation when doing a “gulp build” with an error.

Reply:
should work now. that was a @types/node issue.